### ISSUE TITLE
Beacon pop-up scrolling

### ISSUE THREAD

**Issue by Fitblip** [airgap-it/beacon-sdk#176]
Hi friends,

When the main beacon window pops up, it doesn't have an ability to scroll which makes it harder to click/find stuff. 

If you add in the following to `beacon-alert__base`:
```css
overflow: scroll;
```

And set `beacon-modal__wrapper` to:
```css
position: relative;
top: 0;
display: inline-block;
z-index: 2147483000;
max-width: 500px;
width: 80%;
padding: 24px;
```

The model can scroll freely and users can see things regardless of their screen size. Please note: I haven't tested this on mobile!

---

### Before:

https://user-images.githubusercontent.com/1072598/111562184-21e7f700-876c-11eb-8d9e-9e158d32b0cc.mov

### After:

https://user-images.githubusercontent.com/1072598/111562629-e13cad80-876c-11eb-84fb-6f3f34197645.mov

**Comment by pascuin** [comment]
Thanks for reporting this, we'll look into it.

**Comment by AndreasGassmann** [comment]
Included in 2.2.3
